client/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -21,8 +21,8 @@ describe('CodeEvalResultObjectFactory', function() {
   var codeEvalResult;
   var CODE = "code";
   var OUTPUT = "output";
-  var CORRECTNESS_TEST_RESULTS = "correctnessTestResults";
-  var LAST_TASK_RESULTS = "s";
+  var CORRECTNESS_TEST_RESULTS = ["xyz", "xyz", "xyz", "xyz", "xyz"];
```

Comment:
Might as well use different strings, so that you can tell that the last one is actually the last one (and not one of the others)?